### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
…anslations/reviews added/addressed

Signed-off-by: Vansh Bhatia <vansh@chaosnative.com>

## Proposed changes

 - Added CTA to home page cards
 - Added appropriate color feedbacks to workflow states
 - Addressed translation/constants addition
 - Better alignments on home page cards

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts [hunk 1]
```diff
@@ -19,6 +19,10 @@ const useStyles = makeStyles((theme) => ({
       color: theme.palette.primary.main,
       marginRight: theme.spacing(4.5),
     },
+
+    '& img': {
+      margin: theme.spacing(0, 0.25, -0.375, 0),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts:24]: is the negative margin required?

File: litmus-portal/frontend/src/models/graphql/user.ts [hunk 2]
```diff
@@ -127,3 +127,8 @@ export enum UserRole {
   admin = 'admin',
   user = 'user',
 }
+
+export enum InvitationStatus {
+  PENDING = 'Pending',
+  ACCEPTED = 'Accepted',
+}
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/models/graphql/user.ts:134]: Good practice !

**Review by ishangupta-ds — APPROVED** [review]
some normal functions can be converted to arrow functions to maintain consistency rest changes look good ! cc: @vanshBhatia-A4k9

**Review by SarthakJain26 — APPROVED** [review]

File: litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts (hunk 1 above)
Comment (vanshBhatia-A4k9) [reply litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts:24]: For once yes, it is required as litmus-UI button won't allow me to push the img down without expanding the whole button height, I had to override that

**Review by Saranya-jena — APPROVED** [review]

**Comment by vanshBhatia-A4k9** [comment]
Sure @ishangupta-ds as it is an incremental change, will try to refactor it more as you suggested. Merging for now so that people can live test it and give further feedback

File: litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts (hunk 1 above)
Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/Home/AgentConfiguredHome/RecentWorkflowRuns/styles.ts:24]: Cc: @arkajyotiMukherjee @S-ayanide

**Pull request merged**